Tools/jit/template.c

```diff
@@ -91,28 +104,16 @@ _JIT_ENTRY(_PyInterpreterFrame *frame, PyObject **stack_pointer, PyThreadState *
     }
     PATCH_JUMP(_JIT_CONTINUE);
     // Labels that the instruction implementations expect to exist:
-unbound_local_error_tier_two:
-    _PyEval_FormatExcCheckArg(
-        tstate, PyExc_UnboundLocalError, UNBOUNDLOCAL_ERROR_MSG,
-        PyTuple_GetItem(_PyFrame_GetCode(frame)->co_localsplusnames, oparg));
-    goto error_tier_two;
-pop_4_error_tier_two:
-    STACK_SHRINK(1);
-pop_3_error_tier_two:
-    STACK_SHRINK(1);
-pop_2_error_tier_two:
-    STACK_SHRINK(1);
-pop_1_error_tier_two:
-    STACK_SHRINK(1);
+
 error_tier_two:
     tstate->previous_executor = (PyObject *)current_executor;
     GOTO_TIER_ONE(NULL);
-deoptimize:
+exit_to_tier1:
```

Comment:
Note to other reviewers: this is only used in `DEOPT_IF` and friends. It's completely gone in anything else so it doesn't add to template size.